build-tools/dependencies/dependencies.projitems

```diff
@@ -56,6 +56,7 @@
     </RequiredProgram>
     <RequiredProgram Include="$(ManagedRuntime)"    Condition=" '$(ManagedRuntime)' != '' ">
       <MinimumVersion>$(MonoRequiredMinimumVersion)</MinimumVersion>
+      <MaximumVersion Condition=" '$(IgnoreMaxMonoVersion)' == '' " >$(MonoRequiredMaximumVersion)</MaximumVersion>
```

Comment:
This should be `Condition=" '$(IgnoreMaxMonoVersion)' == 'True' "`, not `== ''`, so that if someone sets it to False something "reasonable" happens.